packages/site/formideploy.config.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * Formideploy configuration overrides.
+ */
+module.exports = {
+  lander: {
+    name: 'urql',
+  },
+  production: {
```

Comment:
When we merge, we'll check whether or not this deployed correctly, if it has we can move on to remove this in a follow-up PR.